src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Set Up Logical Data Replication
+summary: Follow a tutorial to set up logical data replication between two clusters.
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+In this tutorial, you will set up **logical data replication (LDR)** between two CockroachDB clusters. Both clusters are active and can serve traffic. You can apply the outlined steps to create a _unidirectional_ LDR stream from a source to a destination (cluster A to cluster B). 
+
+To create a _bidirectional_ LDR stream, you can complete the [optional step](#step-4-optional-set-up-bidirectional-ldr-streams) to start a second LDR stream that sends writes from the table on cluster B to the table on cluster A. In a bidirectional setup, one cluster operates as both a source and a destination in separate LDR jobs. 
+
+{% comment  %}Will link to LDR overview when published for descriptions of use case topology here{% endcomment %}
+
+<image src="{{ 'images/v24.3/bidirectional-stream.svg' | relative_url }}" alt="Diagram showing a bidirectional LDR stream from cluster A to B and back again from cluster B to A." style="width:70%" />
+
+## Tutorial overview
+
+If you're setting up a bidirectional LDR stream, both clusters will act as a source and a destination in the respective LDR streams. The high-level steps are:
```

Comment:
"If you're planning on creating a bidirectional setup with LDR?" Bidirectional replication?

Reply:
I've adjusted to "If you're setting up bidirectional LDR"... I think it would be great to just have "uni/bidirectional LDR" as the description of those setups, so we don't have to describe it in every sentence. I think we can define this more in the technical overview. Lmk if you want to rethink this still.